### DISCUSSION TITLE
Got a blue screen and lost all scripts and data. I assume there is no easy way to get it back?

### DISCUSSION THREAD

**Discussion by TimidScript** [violentmonkey/violentmonkey#1701] [Q&A]
Hi. Got a blue screen and it corrupt VM database. The script list is empty and I assume all the data is lost. Is there a way to get the data back? I cannot install scripts as the database as it is irresponsive. 

Questiom, is the sync also a backup system? So if it crashes and losses everything, can I just resync?

**Comment by tophf** [comment]
Sync is the same as backup, but online and two-way. I guess you can reinstall the extension if the db is indeed corrupted, then configure sync and it'll fetch the data back automatically.
